### PR TITLE
Uncomment values and update to v0.24.1

### PR DESCRIPTION
**Pull request by manuelbuil**
This PR does three things:
* Uncomment values. Otherwise we can't configure them and we need to configure VNI for Windows
* Move to flannel version v0.24.1
* Start using the flannel released tarball

### CODE REVIEW

**Review by thomasferrandiz — APPROVED** [review]

File: packages/rke2-flannel/package.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 url: https://github.com/flannel-io/flannel.git
```
Comment (rbrtbnfgl) [inline packages/rke2-flannel/package.yaml:1]:
```suggestion
url: https://github.com/flannel-io/flannel/releases/download/v0.24.1/flannel.tgz
```

File: packages/rke2-flannel/package.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 subdirectory: chart/kube-flannel
```
Comment (rbrtbnfgl) [inline packages/rke2-flannel/package.yaml:2]:
```suggestion

```

File: packages/rke2-flannel/package.yaml [hunk 3]
```diff
@@ -1,4 +1,4 @@
...
-commit: v0.24.0
-packageVersion: 01
+commit: 4c1b5fb7bc93d5348992a980ad4e8a8e21bdc9aa #v0.24.1
```
Comment (rbrtbnfgl) [inline packages/rke2-flannel/package.yaml:3]:
```suggestion

```

**Review by rbrtbnfgl — CHANGES_REQUESTED** [review]

File: packages/rke2-flannel/package.yaml (hunk 1 above)
Comment (manuelbuil) [reply packages/rke2-flannel/package.yaml]: idea fantastica!

File: packages/rke2-flannel/package.yaml (hunk 2 above)
Comment (manuelbuil) [reply packages/rke2-flannel/package.yaml]: Grande Roberto! ;)

**Review by rbrtbnfgl — APPROVED** [review]

**Pull request merged**